app/javascript/dashboard/routes/dashboard/settings/sla/Index.vue

```diff
@@ -0,0 +1,142 @@
+              <td class="sla-title">
+                <span class="overflow-hidden whitespace-nowrap text-ellipsis">{{
+                  sla.name
+                }}</span>
+              </td>
+              <td>{{ sla.description }}</td>
+              <td>
+                <span class="flex items-center">
+                  {{ sla.first_response_time_threshold }}
+                </span>
+              </td>
+              <td>
+                <span class="flex items-center">
+                  {{ sla.next_response_time_threshold }}
+                </span>
+              </td>
+              <td>
+                <span class="flex items-center">
+                  {{ sla.resolution_time_threshold }}
+                </span>
+              </td>
+              <td>
+                <span class="flex items-center">
+                  {{ sla.only_during_business_hours }}
+                </span>
+              </td>
+              <td class="button-wrapper">
+                <woot-button
+                  v-tooltip.top="$t('SLA.FORM.EDIT')"
+                  variant="smooth"
+                  size="tiny"
+                  color-scheme="secondary"
+                  class-names="grey-btn"
+                  :is-loading="loading[sla.id]"
+                  icon="edit"
+                  @click="openEditPopup(sla)"
+                />
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+
+      <div class="w-[34%]">
+        <span v-dompurify-html="$t('SLA.SIDEBAR_TXT')" />
+      </div>
+    </div>
+    <woot-modal :show.sync="showAddPopup" :on-close="hideAddPopup">
+      <add-SLA @close="hideAddPopup" />
+    </woot-modal>
+
+    <woot-modal :show.sync="showEditPopup" :on-close="hideEditPopup">
+      <edit-SLA :selected-response="selectedResponse" @close="hideEditPopup" />
+    </woot-modal>
+  </div>
+</template>
+<script>
+import { mapGetters } from 'vuex';
+
+import AddSLA from './AddSLA.vue';
+import EditSLA from './EditSLA.vue';
+import alertMixin from 'shared/mixins/alertMixin';
+
+export default {
+  components: {
+    AddSLA,
+    EditSLA,
+  },
+  mixins: [alertMixin],
+  data() {
+    return {
+      loading: {},
+      showAddPopup: false,
+      showEditPopup: false,
+      selectedResponse: {},
+    };
+  },
+  computed: {
+    ...mapGetters({
+      records: 'sla/getSLA',
+      uiFlags: 'sla/getUIFlags',
+    }),
+  },
+  mounted() {
+    this.$store.dispatch('sla/get');
+  },
+  methods: {
+    openAddPopup() {
+      this.showAddPopup = true;
+    },
+    hideAddPopup() {
+      this.showAddPopup = false;
+    },
+    openEditPopup(response) {
+      this.showEditPopup = true;
+      this.selectedResponse = response;
+    },
+    hideEditPopup() {
+      this.showEditPopup = false;
+    },
+  },
+};
+</script>
+
+<style scoped lang="scss">
+@import '~dashboard/assets/scss/variables';
+.sla-title {
+  span {
+    @apply inline-block;
+  }
+}
+</style>
```

Comment:
Let us not use custom css, please use Tailwind classes.

Reply:
done